example/utils.rs

```diff
@@ -3,16 +3,28 @@ use std::fs;
 use std::io::Result;
 use std::path::Path;
 
-pub const SOCK_ADDR: &str = "unix:///tmp/ttrpc-test";
+#[cfg(unix)]
+pub const SOCK_ADDR: &str = r"unix:///tmp/ttrpc-test";
 
+#[cfg(windows)]
+pub const SOCK_ADDR: &str = r"\\.\pipe\ttrpc-test";
+
+#[cfg(unix)]
 pub fn remove_if_sock_exist(sock_addr: &str) -> Result<()> {
     let path = sock_addr
         .strip_prefix("unix://")
         .expect("socket address is not expected");
 
     if Path::new(path).exists() {
```

Comment:
Got warning: unused import: `std::path::Path`, consider to use std::path::Path here and remove the `use std::path::Path;` above